apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -663,4 +671,11 @@ defmodule SiteWeb.TripPlanView do
     |> Itinerary.transit_legs()
     |> Enum.any?(fn leg -> leg_is_from_or_to_airport?(leg) end)
   end
+
+  @spec itinerary_contains_capeflyer?(Itinerary.t()) :: boolean()
```

Comment:
Should both `itinerary_is_from_or_to_airport` and `itinerary_contains_capeflyer` be handled the same way?  Either as a function that takes itinerary, or as an assigned property in the render function

Reply:
Good point -- I can probably refactor this.